packages/dashboard/lib/DashboardServer.ts

```diff
@@ -76,8 +101,11 @@ export class DashboardServer {
 
     this.expressApp.get("/ports", this.getPorts.bind(this));
 
+    this.subscribeSocket = await this.connectToSubscribePort();
+
     if (this.rpc) {
```

Comment:
This probably is legacy code that should be removed. In what circumstances would a user want RPC disabled here?

Reply:
Removed.